src/renderer/components/Cards/ValidatorStat.tsx

```diff
@@ -4,29 +4,37 @@ import {utils} from "ethers";
 export interface IValidatorStatProps {
     title: string;
     type: string;
-    value: bigint | number | string;
+    value?: bigint | number | boolean;
 }
 const renderROI = (props: IValidatorStatProps): React.ReactElement => {
     return (
         <div className='validator-card-container'>
             <h5>{props.title}</h5>
-            <h1 className={props.value >= 0 ? "plus" : "minus"}>
-                {props.value >= 0 ? "+" : ""}
-                {props.value}%
-            </h1>
-            <h5 className={props.value >= 0 ? "plus" : "minus"}>{props.type}</h5>
+            {props.value === undefined ? (
+                <h1>N/A</h1>
+            ) : (
+                <h1 className={props.value >= 0 ? "plus" : "minus"}>
+                    {props.value >= 0 ? "+" : ""}
+                    {props.value}%
```

Comment:
shoudln't this be formatted to be rounded to two decimals?

Reply:
dunno just extended to display `N/A` if there is no value

Reply:
let's format it to be on safe side

Reply:
i check now and yeah its formatted to 2 decimals at `calculateROI` function
https://github.com/NodeFactoryIo/ChainGuardian/blob/78743d0b0d429ea60e5c2745d081274762cb9c67/src/renderer/services/utils/math.ts#L16